monty/json.py

```diff
@@ -680,24 +895,12 @@ def jsanitize(
         return obj
     if isinstance(obj, (list, tuple)):
         return [
-            jsanitize(
-                i,
-                strict=strict,
-                allow_bson=allow_bson,
-                enum_values=enum_values,
-                recursive_msonable=recursive_msonable,
-            )
+            jsanitize(i, strict=strict, allow_bson=allow_bson, enum_values=enum_values)
```

Comment:
I should have reviewed this better, but the `recursive_msonable` kwarg was removed in a few places. I'll make a patch.